packages/core/test/invoke.test.ts

```diff
@@ -2363,3 +2363,50 @@ describe('invoke', () => {
     });
   });
 });
+
+describe('services option', () => {
+  it('should resolve service from options with string src', (done) => {
```

Comment:
this test title doesn't seem right for what we are testing here, maybe something like that?
```suggestion
  it('should provide data params to a service creator', (done) => {
```